lib/trino-geospatial-toolkit/src/main/java/io/trino/geospatial/GeometryUtils.java

```diff
@@ -187,48 +176,13 @@ public static boolean isPointOrRectangle(OGCGeometry ogcGeometry, Envelope envel
     public static org.locationtech.jts.geom.Geometry jtsGeometryFromJson(String json)
     {
         try {
-            org.locationtech.jts.geom.Geometry emptyGeoJsonOverride = getEmptyGeometryOverride(json);
-            if (emptyGeoJsonOverride != null) {
-                return emptyGeoJsonOverride;
-            }
             return new GeoJsonReader().read(json);
         }
         catch (ParseException | IllegalArgumentException e) {
             throw new TrinoException(INVALID_FUNCTION_ARGUMENT, "Invalid GeoJSON: " + e.getMessage(), e);
         }
     }
 
-    /**
-     * Return an empty geometry in the cases in which the locationtech library
-     * doesn't when the coordinates attribute is an empty array. In particular,
-     * these two cases are handled by the underlying library as follows:
-     * {type:Point, coordinates:[]} -> POINT (0 0)
-     * {type:Polygon, coordinates[]} -> Exception during parsing
-     * To circumvent these inconsistencies, we catch this upfront and return
-     * the correct empty geometry.
-     * TODO: Remove if/when https://github.com/locationtech/jts/issues/684 is fixed.
-     */
-    private static org.locationtech.jts.geom.Geometry getEmptyGeometryOverride(String json)
-    {
-        try {
-            JsonNode jsonNode = OBJECT_MAPPER.readTree(json);
-            JsonNode typeNode = jsonNode.get(TYPE_ATTRIBUTE);
-            if (typeNode != null) {
-                org.locationtech.jts.geom.Geometry emptyGeometry = EMPTY_ATOMIC_GEOMETRY_OVERRIDE.get(typeNode.textValue());
-                if (emptyGeometry != null) {
-                    JsonNode coordinatesNode = jsonNode.get(COORDINATES_ATTRIBUTE);
-                    if (coordinatesNode != null && coordinatesNode.isArray() && coordinatesNode.isEmpty()) {
-                        return emptyGeometry;
-                    }
-                }
-            }
-        }
-        catch (JsonProcessingException e) {
-            // Ignore and have subsequent GeoJsonReader throw
-        }
-        return null;
-    }
-
```

Comment:
The subsequent issue (https://github.com/locationtech/jts/issues/411) was also fixed. Can we update the method as well?